src/main/java/edu/greenblitz/tobyDetermined/subsystems/swerve/SdsSwerveModule.java

```diff
@@ -184,15 +184,15 @@ public SwerveModuleState getModuleState() {
 	}
 	
 	@Override
-	public boolean isAtAngle(double targetAngleInRads, double errorInRads) {
-		double currentAngleInRads = getModuleAngle() % Math.PI;
-		targetAngleInRads = targetAngleInRads % Math.PI;
-		boolean isInRange = false;
-		for (int i = -1; i <= 1 ; i++) {
-			isInRange |= (currentAngleInRads +Math.PI*i < targetAngleInRads + errorInRads
-					&& currentAngleInRads +Math.PI*i > targetAngleInRads - errorInRads);
-		}
-		return isInRange;
+	public boolean isAtAngle(double targetAngleInRads, double tolerance) {
+		
+		double currentAngleInRads = getModuleAngle();
+		return (currentAngleInRads - targetAngleInRads) %(2*Math.PI) < tolerance
```

Comment:
See my comment on the previous swerve, you just proved my point.
Also, why is this function written twice?

Reply:
Raz: After more thought, I think that changing this will take to much effort to do during the season and it might be too late for that. For next year I think we need to work more on the structure of the code, as this can cause issues like this.